keyboards/pdxkbc/readme.md

```diff
@@ -0,0 +1,17 @@
+# pdxkbc
+
+![pdxkbc](https://i.imgur.com/GgNvZcW.jpg)
+
+A macropad made for the Portland Keyboard Club and DEF CON
+
+Keyboard Maintainer: [Franklin Harding](https://github.com/yourusername)  
```

Comment:
You might want to set `yourusername` to **fharding1**.

Reply:
Heh, thanks.